dataprofiler/profilers/profile_builder.py

```diff
@@ -504,7 +511,33 @@ def _update_row_statistics(self, data):
         self.hashed_row_dict = dict.fromkeys(
             pd.util.hash_pandas_object(data, index=False), True
         )
-        self.null_in_row_count = data.isnull().any(axis=1).sum()
+
+        # Calculate Null Column Count
+        null_rows = {}
+        null_in_row_count = {}
+        first_col_flag = True
+        for column in self.profile:
+            if "statistics" in self.profile[column].profile and \
+                    "null_types_index" in self.profile[column]\
+                    .profile["statistics"]:
```

Comment:
are there cases where this was untrue? Is it possible to turn off in options?

Reply:
Instead of calling from the profile, we can use the internals which will be cheaper than building an entire profile, 
i.e.: `self._profile[column].null_types_index`

Reply:
I would assume then we wouldn't need the if statement, i maybe wrong.

Reply:
1. There were cases in the tests where it caused errors but using the internals got rid of the if statement :)